DemoApp/Sources/Views/CustomCallView.swift

```diff
@@ -9,11 +9,11 @@ import SwiftUI
 struct CustomCallView<Factory: ViewFactory>: View {
     
     @Injected(\.colors) var colors
-    
+
     var viewFactory: Factory
     @ObservedObject var viewModel: CallViewModel
-    
-    @StateObject var microphoneChecker = MicrophoneChecker()
+
+    @StateObject var microphoneChecker = InjectedValues[\.microphoneChecker]
```

Comment:
Should actually be an `@ObservedObject` in this case, since the view doesn't own it anymore.